### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by foochifa**
## Problem

Our readme page on github is not updated and contains deprecated commands that will not run as intended.

## Solution

Update our github readme to reflect the process of running forms locally.

**Breaking Changes** 

- No - this PR is backwards compatible  

**Improvements**:

- Updated FormSG title to use logo and a link instead of plain text

### CODE REVIEW

**Comment by foochifa** [comment]
Hey @LinHuiqing do help to add in stuffs for the M1 Chip things, can probably make a sub-section after the running locally portion!

**Review by LinHuiqing — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# FormSG
+<p align="left">
```
Comment (justynoh) [inline README.md:1]: can we get a transparent image instead? 🙏

**Comment by justynoh** [comment]
other than that, lgtm!

**Review by justynoh — APPROVED** [review]

**Pull request merged**